modules/custom/d_p_gallery/templates/paragraph--d-p-gallery.html.twig

```diff
@@ -55,7 +55,12 @@ view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
       <div class="content-wrapper">
         <div class="container">
           {% block content %}
-            {{ content }}
+            {{ content | without('field_d_cta_link') }}
```

Comment:
Nie wiem jak tam inni robią, ale jak dla mnie to pipe zawsze bez spacji :-)
Więc `content|without` 
zobacz jak wygląda w innych szablonach, zwłaszca że poniżej render już zpipowałeś też bez spacji

Reply:
W sumie masz rację, dla mnie też tak jest. Poleciałem copy paste bo gdzieś już tak było i nawet nie zauważyłem.